src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj

```diff
@@ -36,6 +36,10 @@
     <PackageReference Include="System.Threading.AccessControl" Version="8.0.0-preview.2.23128.3" />
     <PackageReference Include="System.Drawing.Common" Version="8.0.0-preview.2.23128.3" />
     <PackageReference Include="NJsonSchema" Version="10.8.0" />
+    <!-- NJsonSchema brings in an old version of CSharp
+      So, we are forcing a newer version.
+    -->
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
```

Comment:
With .NET SDK 8.0, we are supposed to use the `Microsoft.Csharp` that is shipped with .NET 8:

![image](https://user-images.githubusercontent.com/127450/232635960-46a75b0d-c9c3-40b2-aeaf-7b76d23cab06.png)

Will this change brings the old 4.7.0 version of the assembly instead?

Reply:
No

Reply:
Then I'm not sure about the purpose of this change. The `Microsoft.CSharp.dll` appearing in the publish folder without this change is 8.0.0.0, which means `NJsonSchema` doesn't bring in an old version of CSharp for PowerShell build.

Reply:
Chatted with Travis offline, and updated the PR description.